assets/js/linear_algebra/dot_product_collide.js

```diff
@@ -15,17 +15,17 @@ let origin = [150, 140],
     svg = null,
     uTvv_opacity = 0;
 
-let w_unit = 1.0, h_unit = 1.0,
-    dws_array = [0.4, 0.7, 0.7, 0.7],
-    dhs_array = [0.4, 0.4, 0.4, 0.4];
+let w_unit = 1.0, h_unit = 1.0;
 
-let start_coord_x=(375 - origin[0])/scale, 
-    start_coord_y=(75 - origin[1])/scale;
+let start_coord_x=(380 - origin[0])/scale, 
+    start_coord_y=(75 - origin[1])/scale,
+    last_col_coord = start_coord_x + 1.9 * w_unit,
+    last_row_coord = start_coord_y + 0.775 * h_unit;
 
-let bot_right_x = start_coord_x + dws_array.reduce(
-    function(total, d) {return total + d * w_unit;}, 0),
-    bot_right_y = start_coord_y + dhs_array.reduce(
-    function(total, d) {return total + d * h_unit;}, 0);
+let v_cell = {text: 'v =', x: last_col_coord,
+              y: start_coord_y - 0.7 * h_unit, key: 'v'},
+    uT_cell = {text: 'u\u1d40 =', x: start_coord_x - 0.6 * w_unit,
+               y: last_row_coord + 0.09 * h_unit, key: 'u' };
```

Comment:
can we have a variable for these constants 0.7, 0.6, 0.09

Reply:
done